src/common/components/clusterConfiguration/DownloadIso.tsx

```diff
@@ -45,19 +40,29 @@ const DownloadIso: React.FC<DownloadISOProps> = ({
     <>
       <ModalBoxBody>
         <Stack hasGutter>
+          <StackItem>
+            <Alert isInline variant="success" title={t('ai: Discovery ISO is ready for download')}>
+              <>
+                &nbsp;
```

Comment:
is this needed ?

Reply:
Yes, it's part of that: 
![image](https://user-images.githubusercontent.com/69599321/178697915-f717fe4b-65b8-4d6b-bfb9-2402cbf85a6f.png)